ui/build.gradle

```diff
@@ -32,12 +32,8 @@ dependencies {
 }
 
 ext {
-    bintrayName = 'TextInputLayout'
-    publishedArtifactId = 'TextInputLayout'
-
-    libraryName = 'TextInputLayout'
-    libraryDescription = 'Google Material Component without the hint cut out.'
+    PUBLISH_ARTIFACT_ID = 'TextInputLayout'
+    POM_DESCRIPTION = 'TextInputLayout'
```

Comment:
Please provide the previous proper description here

Reply:
Sorry, it got lost somehow, updated
https://github.com/AckeeCZ/TextInputLayout/pull/12/commits#:~:text=%E2%99%BB%EF%B8%8F,and%20fix%20version